### PR TITLE
feat: add pronoun functionality

### PR DESCRIPTION
**Pull request by tenbradley**
## Context:
Adding the ability for pronouns to be added to the profile card. Still needs to be linked to appropriate hypermedia api.

## Quality:
Tests using bsi and local lms by initializing pronouns var in user-profile-card-controller.js. When undefined or empty string, no pronouns are present on card. When a non-empty string, pronouns are present on card.

### CODE REVIEW

**Comment by tenbradley** [comment]
newProfileCard

File: user-profile-card-controller.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ export class UserProfileCardController {
 			const orgDefinedIdEntity = enrolledUserEntity.entity.getSubEntityByRel(Rels.orgDefinedId);
 			const pagerEntity = enrolledUserEntity.entity.getSubEntityByRel(Rels.pager);
 			const userProfileEntity = enrolledUserEntity.entity.getSubEntityByRel(Rels.userProfile);
+			const userPronounEntity = enrolledUserEntity.entity.getSubEntityByRel('pronouns-rel-goes-here');
```
Comment (lloydwaddell) [inline user-profile-card-controller.js:32]: I have a PR up to add the userPronouns rel for this:
https://github.com/Brightspace/d2l-hypermedia-constants/pull/141

File: karma.sauce.conf.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ module.exports = config => {
 			singleRun: true,
 			client: {
 				mocha: {
-					timeout : 5000 // default 2000
+					timeout : 6000 // default 2000
```
Comment (lloydwaddell) [inline karma.sauce.conf.js:59]: required to make accessibility tests pass

**Review by zinabat — APPROVED** [review]

File: user-profile-card-controller.js [hunk 3]
```diff
@@ -6,6 +6,20 @@ export const UserProfileCardControllerErrors = {
 	INVALID_TYPE_BASE_HREF: 'D2L-Labs-Profile-Card: Invalid User HREF Type'
 };
 export class UserProfileCardController {
+
+	static getDefaultProfileCardSettings() {
```
Comment (lloydwaddell) [inline user-profile-card-controller.js:10]: nit: this doesn't need to be static anymore. Originally planned to set this default when the card was being constructed, but ended up changing to the if/else approach here:
https://github.com/BrightspaceUILabs/user-profile-card/pull/71/files#diff-d5a3c24575b90b75e91f2825af4b21d61556b9b648923ea3e86c67a29a0f8c7cR446

**Pull request merged**